### ISSUE TITLE
Have GBSA models been benchmarked on FreeSolv?

### ISSUE THREAD

**Issue by jchodera** [MobleyLab/FreeSolv#24]
@davidlmobley: Have you benchmarked various GBSA models on the FreeSolv dataset yet?  We're just wondering what the expected RMSEs are here.

**Comment by davidlmobley** [comment]
Only on small pieces of it. This is one reason why we want Yank to work for
solvation free energy calculations as we'd love to do this, but we've been
waiting on Yank.

On Mon, Oct 12, 2015 at 12:42 PM, John Chodera <notifications@github.com>
wrote:

> @davidlmobley <https://github.com/davidlmobley>: Have you benchmarked
> various GBSA models on the FreeSolv dataset yet? We're just wondering what
> the expected RMSEs are here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/choderalab/FreeSolv/issues/24>.
>



-- 
David Mobley
dmobley@gmail.com
949-385-2436

**Comment by davidlmobley** [comment]
Closing this for now as we did this recently on Orion for at least one GBSA model. Also, my 2007/2008 work did it in several models for a large chunk of FreeSolv (the original "504 molecule set").

**Comment by jchodera** [comment]
> Closing this for now as we did this recently on Orion for at least one GBSA model. Also, my 2007/2008 work did it in several models for a large chunk of FreeSolv (the original "504 molecule set").

The original origin of this question was that we were getting crazy big errors for applying the GBSA models in OpenMM to the expanded FreeSolv set. It was unclear if these were due to the addition of crazy SAMPL molecules after the 504 molecules set or issues with OpenMM's GBSA implementation. Testing the 504-molecule set with gromacs and the whole set with OpenMM's GBSA doesn't allow us to address the origin of this issue, so it would still be useful to compute them all with gromacs at some point.

**Comment by davidlmobley** [comment]
Ah, OK. I certainly don't mind having that done, but it's not a priority on this end at this point. Though if you had some isolated molecules you wanted checked we could do that easily. Basically we have nothing going on with implicit solvent models at all on our end at present.

@camizanette at one point did some work on looking into some of the largest discrepancies between explicit and implicit solvent in GROMACS and (a) validated our prior results in general, and (b) found that some of these were due to real limitations in the GB models (in some cases, producing POSITIVE solvation free energies for polar compounds!). I believe she's working towards writing this up but it's been delayed by other things which have come up. If any of that info is useful to you we could dig it up -- but again this is on a small subset of FreeSolv.
